2020/13xxx/CVE-2020-13671.json

```diff
@@ -1,18 +1,83 @@
 {
```

Comment:
MISC or CONFIRM, but name != url for reference.

Reply:
I think removing the `name` field will resolve this issue.